error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/StreamRulesTestInput.java

```diff
@@ -322,4 +323,12 @@ Stream<Integer> testStreamOf4() {
   Stream<Integer> testStreamOf5() {
     return ImmutableList.of(1, 2, 3, 4, 5).stream();
   }
+
+  Stream<String> testIterableStream() {
+    return StreamSupport.stream(List.of("foo", "bar").spliterator(), false);
```

Comment:
We prefer immutability
```suggestion
    return StreamSupport.stream(ImmutableList.of("foo", "bar").spliterator(), false);
```

Reply:
I should have phrased this better: We prefer Guava's immutable library.

Reply:
Makes sense!